src/api/get-installations.js

```diff
@@ -9,6 +9,13 @@ import ALIASES from '../aliases';
 var installationsCache = null;
 
 
+async function execWithCodePage (command, codePage) {
```

Comment:
We can move it to `src/utils/exec`. Also, I don't think we need to pass `codePage` around. `65001` is UTF8 codepage (see: https://msdn.microsoft.com/ru-ru/library/windows/desktop/dd317756(v=vs.85).aspx). We can name function `execWinShellUtf8` or smthg like that. \cc @VasilyStrelyaev